tests/suites/0_stateless/09_0008_fuse_optimize_table.sql

```diff
@@ -0,0 +1,55 @@
+DROP DATABASE IF EXISTS db_09_0008;
+CREATE DATABASE db_09_0008;
+USE db_09_0008;
+
+create table t(a uint64);
+
+---------------------------
+
+insert into t values (5);
+insert into t values (6);
+insert into t values (7);
+
+optimize table t compact;
+
+-- expects 4 history items, 3 of previous insertion, 1 for last compaction
+select count(*) from fuse_history('db_09_0008', 't');
+
+---------------------------
+
+-- purge data and history
+optimize table 't' purge;
```

Comment:
`optimize table 't'` equals `optimize table 't' purge`?

Reply:
yes, 

line 20:
```
-- purge data and history
optimize table 't' purge;
-- expect 1 snapshot left
select count(*) from fuse_history('db_09_0008', 't');
```

- wondering why there is a snapshot left? yes, it is not consistent with the behavior of table version 0 has no snapshot file. IMO, later we should save a snapshot for table version 0 as well.

the snapshot left references no segments or blocks. (checked in unit test)

- wondering why the default behavior is purge, but not `compact and purge`?
   currently, the impl of `compact` is brute-force or ... dumb, how about changing the default behavior of `optimize` to `compact & purge` till the impl of `compact` is tuned